### ISSUE TITLE
Control.Geocoder.js:161 Uncaught TypeError: this.options.geocoder[mode] is not a function

### ISSUE THREAD

**Issue by zopyx** [perliedman/leaflet-control-geocoder#200]
Using the geocoder plugin gives an error.

See https://dev.itad.de/information/anlagen

```
Control.Geocoder.js:161 Uncaught TypeError: this.options.geocoder[mode] is not a function
    at e._geocode (Control.Geocoder.js:161)
    at e.<anonymous> (Control.Geocoder.js:92)
    at r (leaflet.js:5)
    at j (leaflet.js:5)
    at leaflet.js:5

```

**Comment by zopyx** [comment]
Related

https://github.com/perliedman/leaflet-control-geocoder/issues/157

**Comment by perliedman** [comment]
@zopyx hi, it's not clear to me how I can reproduce this problem - I tried geocoding from the link you provided, and it looks like it's working as expected, and I have no errors related to this in the console.

Also, which version are you running?

**Comment by zopyx** [comment]
I updated the local code with this fix

https://github.com/perliedman/leaflet-control-geocoder/pull/184

which does not seem to be part of the latest release.

**Comment by perliedman** [comment]
Huh, looks like we should make a new release. Thanks for the heads up!

**Comment by idenchev** [comment]
Hello,

I am experiencing the same issue. As mentioned in other threads it happens only when **collapsed: false**

**Comment by idenchev** [comment]
Thanks, zopyx :) In fact #184 refers to #157 and what is shown there is not clear to me - it is not specified what exactly to change. Also the code screenshots differ from the code in current release. Let's wait for perliedman for a new release. Actually this problem does not bother me since functionallity is perfect :)

**Comment by perliedman** [comment]
Closing, since #184 is merged and #157 closed. Please give a shout if I have misunderstood the situation.
